core/src/main/java/io/grpc/internal/ServiceConfigUtil.java

```diff
@@ -310,8 +313,8 @@ static Integer getMaxResponseMessageBytesFromMethodConfig(Map<String, Object> me
           {"xds" :
             {
               "balancerName": "balancer1",
-              "childPolicy": {...},
-              "fallbackPolicy": {...},
+              "childPolicy": [...,]
+              "fallbackPolicy": [...],
```

Comment:
[...] => [...,]

I know it's just a comment, but still...

Reply:
Fixed. Thanks for the catch.